src/components/atoms/Notification.tsx

```diff
@@ -0,0 +1,47 @@
+import { Notification, Icon } from "@ledgerhq/react-ui";
+import { CustomContentProps, SnackbarContent } from "notistack";
+import React from "react";
+
+// NOTE: https://notistack.com/features/customization#custom-variant-(typescript)
+declare module "notistack" {
+  // eslint-disable-next-line @typescript-eslint/consistent-type-definitions
+  interface VariantOverrides {
+    errorNotification: {
+      errorType: string;
+    };
+  }
+}
+
+const warningBadge = (
+  <Notification.Badge
+    color="warning.c50"
+    backgroundColor="warning.c100"
+    size={26}
+    icon={<Icon name="Warning" size={20} />}
+  />
+);
+
+type ErrorNotificationProps = CustomContentProps & {
+  errorType: string;
+};
+
+export const ErrorNotification = React.forwardRef<
+  HTMLDivElement,
+  ErrorNotificationProps
+>((props, ref) => {
+  const { id, errorType, message } = props;
+
+  return (
+    <SnackbarContent ref={ref} key={id}>
```

Comment:
From the best practices doc they suggest to use the style too

Also the key might not be needed

I added the `role="alert"` as seen in the doc and also to follow the aria recommendations
```suggestion
  const { id, errorType, message, ...other } = props;

  return (
    <SnackbarContent ref={ref} role="alert" {...other}>
```